File: SkyPayment.Shared/Menu/MenuCreateModel.cs
using System.Collections.Generic;


namespace SkyPayment.Shared
{
    public class MenuCreateModel
    {
        public string Name { get; set; }
        public ICollection<MenuItemCreateModel> Items { get; set; }
        public IEnumerable<string> RestaurantId { get; set; }
        public string ManagerId { get; set; }
    }

    
}

File: SkyPayment.UI/skypayment-management-ui/src/api-models/menu/MenuCreateModel.ts
export interface MenuCreateModel {
    name: string;
    items: MenuItemCreateModel[];
    restaurantId: string[];
    managerId: string;
}

export interface MenuItemCreateModel {
    name: string;
    price: number;
    ingredients: string;
    productContent: ProductContent;
    isActive: boolean;
}
export enum ProductContent {
    IsVegan = 1,
    IsDiabetic = 2,
    IsGlutenFree = 4,
    IsVegetarian = 8
}
